hazelcast/src/test/java/com/hazelcast/client/config/YamlClientConfigImportVariableReplacementTest.java

```diff
@@ -50,15 +51,42 @@
     public ExpectedException rule = ExpectedException.none();
 
     @Override
-    @Test(expected = InvalidConfigurationException.class)
-    public void testImportElementOnlyAppersInTopLevel() {
+    @Test
+    public void testImportElementOnlyAppearsInTopLevel() throws IOException {
+        File config1 = createConfigFile("hz1", ".yaml");
+        FileOutputStream os1 = new FileOutputStream(config1);
+        String config1Yaml = ""
+                + "hazelcast-client:\n"
+                + "  instance-name: my-instance";
+        writeStringToStreamAndClose(os1, config1Yaml);
         String yaml = ""
-                + "hazelcast:\n"
+                + "hazelcast-client:\n"
                 + "  network:\n"
                 + "    import:\n"
-                + "      resource: \"\"";
+                + "      - file:///" + config1.getAbsolutePath() + "\"";
 
-        buildConfig(yaml);
+        ClientConfig clientConfig = buildConfig(yaml);
+        assertNotEquals("my-instance", clientConfig.getInstanceName());
```

Comment:
Minor: may it would be better to check which value is expected here?

Reply:
It would be better, I change this to `assertNull` instead, since in this case `instanceName` is not expected to be set. I'll change this to `assertNull` instead.

Reply:
Done